install/mysql/glpi-empty.sql

```diff
@@ -2956,6 +2956,8 @@ CREATE TABLE `glpi_entities` (
   `date_mod` timestamp NULL DEFAULT NULL,
   `date_creation` timestamp NULL DEFAULT NULL,
   `autofill_decommission_date` varchar(255) COLLATE utf8_unicode_ci NOT NULL DEFAULT '-2',
+  `enable_custom_css` int(11) NOT NULL DEFAULT '-2',
```

Comment:
why `-2`?

Reply:
I guess we should use a boolean here, can't we?

Reply:
`-2` (https://github.com/glpi-project/glpi/blob/9.5/bugfixes/inc/entity.class.php#L56) is the value used to defines that configuration is inherited from parent. The `Entity::getUsedConfig()` ( https://github.com/glpi-project/glpi/blob/9.5/bugfixes/inc/entity.class.php#L2338 ) method will recursivly search in parents and returned the first value that is not `-2`.

Reply:
ok :)